server/schedule/storelimit/store_limit.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storelimit
+
+import (
+	"time"
+
+	"github.com/juju/ratelimit"
+)
+
+const (
+	// SmallRegionThreshold is used to represent a region which can be regarded as a small region once the size is small than it.
+	SmallRegionThreshold int64 = 20
+)
+
+// RegionInfluence represents the influence of a operator step, which is used by store limit.
+var RegionInfluence = map[Type]int64{
+	RegionAdd:    1000,
+	RegionRemove: 1000,
```

Comment:
I think the influence of removing a peer should be different from adding a peer.
/cc @nolouch

Reply:
How do you get the influence of adding a peer?

Reply:
> I think the influence of removing a peer should be different from adding a peer.
> /cc @nolouch

how do you get the influence of adding a peer?

Reply:
maybe need some tests.